helpers/presentation_objects.py

```diff
@@ -239,7 +239,7 @@ def format_title(name):
 
     Slight variance on _title() in iris/quickplot.py.
     """
-    title = name.replace("_", " ").title()
+    title = name.replace("_", " ")
```

Comment:
@uwefladrich I _think_ this call to `replace()` comes from an older version of OIFS where the long names in the output files contained underscores. Do you know if the current NEMO/OIFS output has underscores in the long names somewhere? Otherwise we can remove this, as well as l. 255

Reply:
Yes, I have found some `long_name`s with underscores in NEMO. These are probably cases were we need to see over the metadata of variables and in the end they might get better/correct `long_name`s, but currently we have them.

Reply:
So should we keep this line for now, then?

Reply:
Is there a technical reason to remove the underscores, or just aesthetics?

On one hand, if the user has given a `long_name` with underscores, we could say that this is what he gets and therefore remove the line. However, I wonder if not the `standard_name` is a fallback for the title. In that case, `standard_name`s have to have underscores (no spaces) and therefore it might be justified to replace underscores with spaces.

Reply:
It's just aesthetics - and again, the question is how much we manually modify the metadata of the output. But I cannot really think of an example where it makes sense to keep the underscores, so I'd say we leave the line in for now